service/round.go

```diff
@@ -42,6 +43,14 @@ var (
 		Name:      "leaves_total",
 		Help:      "Number of generated leaves in a round",
 	}, []string{"id"})
+
+	batchWriteLatencyMetric = promauto.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "poet",
+		Subsystem: "round",
+		Name:      "batch_write_latency_seconds",
+		Help:      "Latency of batch write operations",
+		Buckets:   prometheus.ExponentialBuckets(0.0001, 1.5, 20),
```

Comment:
i think it can start at 10ms (0.01s). otherwise there will be a lot of wasted metrics

Reply:
I think 10ms is a little high as I observe latencies < 10ms. Changed min to 1ms.